content/actions/creating-actions/publishing-actions-in-github-marketplace.md

```diff
@@ -64,6 +64,15 @@ To remove a published action from {% data variables.product.prodname_marketplace
 1. Select **Publish this action to the {% data variables.product.prodname_marketplace %}** to remove the check from the box.
 1. Click **Update release** at the bottom of the page.
 
+## Transferring action repositories
+
+Action repositories can be moved by exporting or cloning the original repository, making a new repository on the target platform, and then pushing the code to the new repository.
+
+- A repo can be transferred and actions will continue to work because there is a redirect in place (users of the action won't have to update their workflows)
```

Comment:
```suggestion
When a repository admin transfers an action repository, {% data variables.product.prodname_dotcom %} automatically creates a redirect from the previous URL to the new URL, meaning workflows that use the affected action do not need to be updated.
```

I made some edits here to use active voice and adhere to our [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#terms-to-use-or-avoid).